azurepipelines/e2e_test/scenarios/ubuntu-20.04-arm64/vm_setup/devicesetup.py

```diff
@@ -0,0 +1,83 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import json
+import sys
+
+# Note: the intention is that this script is called like:
+# python ./<scenario-name>/vm-setup/devicesetup.py
+sys.path.append('./scenarios/')
```

Comment:
adding to the path seems like a code smell.  is there no better way to accomplish what you need to do?

Reply:
Not for code you're not installing as a module. 

Another method we could use is creating a *.wheel file for the toolkit and then install it as apart of the requirements.txt file.